packages/reports/tests/unit/serializers/report-test.js

```diff
@@ -102,98 +114,87 @@ module('Unit | Serializer | Report', function(hooks) {
+                cid: 'c2',
+                field: 'revenue',
                 parameters: {
-                  currency: 'EUR',
-                  as: 'm2'
-                }
+                  as: 'm2',
+                  currency: 'EUR'
+                },
+                type: 'metric'
+              },
+              {
+                alias: null,
+                cid: 'c3',
+                field: 'property',
+                parameters: {
+                  field: 'id'
+                },
+                type: 'dimension'
               }
             ],
-            dimensions: [{ dimension: 'property' }],
-            filters: [],
-            having: [],
-            sort: [],
-            intervals: [
+            dataSource: 'bardOne',
+            filters: [
               {
-                end: '2018-02-16 00:00:00.000',
-                start: '2018-02-09 00:00:00.000'
+                field: 'network.dateTime',
+                operator: 'bet',
+                parameters: {
+                  grain: 'day'
+                },
+                type: 'timeDimension',
+                values: ['2015-10-02', '2015-10-14']
               }
             ],
-            bardVersion: 'v1',
-            requestVersion: 'v1',
-            dataSource: 'bardOne'
+            limit: null,
+            requestVersion: '2.0',
+            sorts: [],
+            table: 'network'
           },
+          title: 'RequestV2 mmuti-param testing report',
           visualization: {
-            type: 'table',
-            version: 1,
             metadata: {
-              columns: [
-                {
-                  attributes: {
-                    name: 'dateTime'
-                  },
-                  type: 'dateTime',
-                  displayName: 'Date'
-                },
-                {
-                  attributes: {
-                    name: 'property'
-                  },
-                  type: 'dimension',
-                  displayName: 'Property'
+              columnAttributes: {
+                c1: {
+                  canAggregateSubtotal: false
                 },
-                {
-                  attributes: {
-                    name: 'revenue',
-                    parameters: {
-                      currency: 'USD'
-                    }
-                  },
-                  type: 'metric',
-                  displayName: 'Revenue (USD)'
+                c2: {
+                  canAggregateSubtotal: false
                 },
-                {
-                  attributes: {
-                    name: 'revenue',
-                    parameters: {
-                      currency: 'EUR'
-                    }
-                  },
-                  type: 'metric',
-                  displayName: 'Revenue (EUR)'
+                c3: {
+                  canAggregateSubtotal: false
                 }
-              ]
-            }
+              },
+              showTotals: []
```

Comment:
Fix any `showTotals: []` you find
```suggestion
              showTotals: {}
```